optaweb-tsp-ui/src/components/TravelingSalesmanProblem.tsx

```diff
@@ -14,39 +14,47 @@
  * limitations under the License.
  */
 import * as React from 'react';
+import { connect } from 'react-redux';
+import { IAppState } from '../store/configStore';
+import { tspOperations } from '../store/tsp';
 import { ILatLng, ITSPRouteWithSegments } from '../store/tsp/types';
 import LocationList from './LocationList';
 import TspMap from './TspMap';
 
-export interface ITravelingSalesmanProblemProps {
```

Comment:
I think that without the **export** of props interface, the component can't be tested using the interface to pass props

Reply:
True, but there is currently no test for this component, so the export is not needed.